fabfile.py

```diff
@@ -183,6 +183,50 @@ def update_vendor_config():
         system("cd %s ; git pull" % config_file_dir)
 
 
+@task
+def docker_fabric_task():
+    """
+    To be launched with fab -c configs/bluenove-server-configs/dev_staging.rc docker_fabric_task
+    """
+    # env.docker_api_version = '1.17.05'
+    env.docker_tunnel_remote_port = 22024
+    from dockerfabric import cli
+    from dockerfabric.utils.files import temp_dir
+    cwd = os.getcwd()
+    if env.docker_piwik:
+        with temp_dir() as remote_tmp:
+            print remote_tmp
+            try:
+                run("git clone git@github.com:siutin/piwik-docker-compose.git")
+                run("cd piwik-docker-compose && docker-compose up -d")
+                run("docker exec -d piwikdockercompose_db_1 mysqldump --result-file=dump.sql")
+            except:
```

Comment:
This try/except block looks a bit weak, maybe rather check for the presence of `piwik-docker-compose` folder, and if it does not exist then do the `git clone` ?